plugins/org.jboss.tools.openshift.express.core/src/org/jboss/tools/openshift/express/internal/core/connection/ExpressConnection.java

```diff
@@ -52,9 +52,19 @@
  */
 public class ExpressConnection extends AbstractConnection {
 
-	private static final String USER_ID =
-			ExpressCoreActivator.PLUGIN_ID + " " + ExpressCoreActivator.getDefault().getBundle().getVersion();
+	private static final String USER_ID = initUserId();
 
+	/*
+	 * Added to resolve testing
+	 */
+	private static String initUserId(){
+		try{
+			return ExpressCoreActivator.PLUGIN_ID + " " + ExpressCoreActivator.getDefault().getBundle().getVersion();
+		}catch(Exception e){
+			//Log?
```

Comment:
catching plain Exception rarely is a good choice. What's possibly being thrown here?
And then to me if this can throw then we shouldnt have this statically initialized.